source/read-from-mongodb.txt

```diff
@@ -42,6 +42,18 @@ Overview
 
          .. include:: /scala/filters.txt
 
+.. important:: Change Stream Schema Inference
+
+   When the {+driver-short+} infers the schema of a data frame 
+   read from a change stream, by default,
+   it will use the schema of the underlying collection rather than the schema
+   of the change stream. To instruct the connector to use the schema of the 
+   change stream, set the ``change.stream.publish.full.document.only`` option 
+   to ``true``.
+
+   For more information on configuring a read operation, see the
+   :ref:`spark-read-conf` guide.
```

Comment:
S: make this link more specific about the change read read options and link here https://docs-mongodbcom-staging.corp.mongodb.com/spark-connector/docsworker-xlarge/docsp-31169-infer-schema/configuration/read/#change-streams

Reply:
that's what i had initially. not sure why i changed